rfcs/text/0000-alternative-planning.md

```diff
@@ -0,0 +1,315 @@
+2) Theme teams are more flexible to adapt their work as circumstances emerge and change, rather than requiring a 
+planning cycle.
+3) It may be easier for developers to move between theme teams as required, rather than being tied to components that
+face varying workloads.
+4) It will be easier to create theme teams to work on cross-cutting DCP objectives, such as metrics of reliability or
+whole-DCP dashboards.
+5) More developers get a chance to lead if there are theme teams.
+6) Theme teams grow their familiarity with the DCP as a whole. There are no longer per-component silos, though one 
+would have to avoid per-theme silos emerging.
+
+Cons:
+1) Communication has to become more asynchronous and/or the overlap between UK and US West Coast day has to be increased
+as much as possible bearing in mind cultural variation and work-life balance. Otherwise, more theme teams may
+mean even more pressure on limited communication bandwidth.
+2) Theme teams may be smaller than per-component teams - developers moving teams may be a political challenge.
+3) Theme teams may be more isolated. For example, if 3 members are in the US and only 1 in the UK.
+4) It's a risky change that requires commitment and adaptation across the entire DCP. To reduce the risk it could be
+done first on an experimental basis with a single team, or even done partially with smaller 'maintainer' teams still
+nurturing the individual components.
+
+#### Best practice is evangelized rather than mandated
+This approach recognizes that there is *cultural variation* between teams. It assumes that imposing a process will not bring
+alignment because there is *no overall control*. Teams also face different circumstances which means one size will not
+fit all. Alignment is valuable but comes about through evangelization, discussion of shared problems and spontaneous convergence.
+
+Pros:
+1) Teams can adapt their workflow to fit their circumstances, both at the present time and as those circumstances 
+change over time.
+2) Alignment is strong because it comes about through choice and agreement.
+
+Cons:
+1) It's more work for project managers since the scope for tracking status automatically is reduced as team
+workflows are less aligned.
+ 
+#### A culture of individual empowerment
+People are afraid to take risks in this project, despite our encouragement. I've seen this many times in my relatively
+short time with the DCP. Why is this? One reason is that people find the de facto mechanism of cross-component change,
+the RFC, extremely daunting. It's:
+
+a) Formally structured so one needs to spend a lot of time writing it.
+b) Going to be reviewed by a lot of people, you need to get it right or you'll look stupid. Both content and form.
+c) Going to generate a metric ton of discussion that will take weeks to understand and wade through.
+d) Get approved but then sit around because implementing it isn't a priority. 
```

Comment:
There is an expectation that RFC(s) will be related to priorities expressed in Product or Technical Objectives which would result in their implementation.

Reply:
I agree that the RFC process can be unwieldy. Unclear how to resolve an issue when one reviewer requests changes that the author doesn't agree with and others haven't commented on. How do we reach "rough consensus" to address feedback?

Reply:
It depends on the specific situation. Is the reviewer offering concrete alternatives that can be compared and contrasted? Did the author(s) address if not accommodate the issue? Do you have a specific example in mind?

In preparation for Oversight review, the Shepherd is supposed to _Summarize the review discussion for the Approvers in the top-level summary comment in the RFC pull request_ to capture such tensions.

Do you have [suggestions](https://github.com/HumanCellAtlas/dcp-community/issues) to improve the RFC process in its next version?

Reply:
> In preparation for Oversight review, t

The fact that Oversight is involved in the RFC process is indicative of a problem with the RFC process.  They are far too distant from technology to be able to provide good decisions.  The advice will come in a vacuum of understanding of intricate technical details of most RFCs.

Oversote's involvement should stop at the roadmap.    If those of us who are technologists are unable to translate the roadmap into technology decisions, we have demonstrated that we are not capable of implementing the DCP.

Reply:
@diekhans - The _Oversight Committee_ is **not** involved in the RFC process. Let me refresh your memory.

When the RFC process was written, an [Oversight Review](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0001-rfc-process.md#shepherding-rfcs) referred to the DCP PM and Technical Architecture teams:

> For software RFCs, request an Oversight review of the RFC on the HumanCellAtlas #tech-architecture slack channel. Include a link to the RFC pull request and the last call deadline
Add the RFC as an agenda item to the next DCP Architecture meeting

> For governance RFCs, request an Oversight review of the RFC on the HumanCellAtlas #dcp-project-mgmt slack channel. Include a link to the RFC pull request and the last call deadline:

When the original _Steering Committee_ reincarnated as the _Oversight Committee_, _Oversight_ became the shorthand for referring to the committee, because _OC_ was already taken by the _Organizing Committee_.  If it's confusing, then perhaps we can adopt a new name for these types of reviews.